src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/request/RecordNomisLicenceReasonRequest.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+
+@Schema(description = "Request object for creating a new NOMIS Time Served licence creation reason")
+data class RecordNomisLicenceReasonRequest(
+  @field:Schema(description = "The prison NOMIS identifier for this offender", example = "A1234AA")
+  @field:NotBlank(message = "nomsId must not be blank")
+  val nomsId: String,
```

Comment:
nickpick: Nomis Is is really an internal legacy system and should be encapsulated,.. we should used prison number,.. because when we don't have Nomis in the future it's has no meaning.